semconv/v1.19.0/resource.go

```diff
@@ -916,6 +916,46 @@ func FaaSMaxMemory(val int) attribute.KeyValue {
 	return FaaSMaxMemoryKey.Int(val)
 }
 
+// Resource used by Google (GCP) Cloud Run
```

Comment:
If you look at the [very top](https://github.com/open-telemetry/opentelemetry-go/blob/d003b70a5c6f1e30a17a7f8dfb5fc619873347a4/semconv/v1.19.0/resource.go#L15) of this file, you will see this is a generated file.

This doesn't do the needed work to add parsing into the tooling to generate these attributes from the actual source.

Reply:
@MadVikingGod yeah sorry, I noticed that after I opened this. Switched it to a draft just so I have a placeholder but I can close it if that's easier.

Will these new resources just be automatically pulled in with the next release? Or will I need to open a PR to generate the files?

(btw, I checked a couple of the readmes and contributing files and it wasn't obvious to me what the workflow for updating these is)